website/docs/create-your-composite.mdx

```diff
@@ -0,0 +1,111 @@
+import Tabs from  '@theme/Tabs'
+import TabItem from  '@theme/TabItem'
+
+# Create your composite
+The second step in getting started with ComposeDB on Ceramic is creating your composite to serve as your database schema. In this guide, we will create your first composite.
+
+
+>💡 Before proceeding you must have [set up your environment](./set-up-your-environment) in the previous step
```

Comment:
Better use an [admonition](https://docusaurus.io/docs/markdown-features/admonitions) here to emphasis the warning I think.